### PR TITLE
fix: add missed translation in uk_UA

### PR DESCRIPTION
**Pull request by andrii-milishchuk**
[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [x] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 💡 Background and solution
1. The DatePicker does not have a lot of translations
2. image
3. Using antd's UK locale will include all new translations

### 📝 Changelog
- Added ukranian translations for datepicker

### CODE REVIEW

**Comment by gas4057** [comment]
Cool bro! SO AMAZING!

File: components/date-picker/locale/uk_UA.ts [hunk 1]
```diff
@@ -1,21 +1,22 @@
-import CalendarLocale from 'rc-picker/lib/locale/uk_UA';
-
+import CalendarLocale from "rc-picker/es/locale/uk_UA";
```
Comment (afc163) [inline components/date-picker/locale/uk_UA.ts:1]:
```suggestion
import CalendarLocale from "rc-picker/lib/locale/uk_UA";
```

**Comment by yachek** [comment]
Nice, very usefull for me in development

**Comment by afc163** [comment]
CI failed.

**Review by li-jia-nan — APPROVED** [review]

**Pull request merged**